internal/envconfig/envconfig_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package envconfig
+
+import (
+	"os"
+	"testing"
+
+	"google.golang.org/grpc/internal/grpctest"
+)
+
+type s struct {
+	grpctest.Tester
+}
+
+func Test(t *testing.T) {
+	grpctest.RunSubTests(t, s{})
+}
+
+func (s) TestUint64FromEnv(t *testing.T) {
+
+	var testCases = []struct {
+		name          string
+		val           string
+		def, min, max uint64
+		want          uint64
+	}{
+		{
+			name: "error parsing; want default",
```

Comment:
Should we change the test names to be in snake_case instead, as the `go test` command would do that anyways, and will `;`s in there, the test name might look weird.

Reply:
I think this is fine IMO.  AFAICT there's no real guidance given on this by the Go team, and the name is essentially just a convenience & informational.  No matter what we name things, they will be copy-pasteable to re-run.  That said, the "want"s are pretty obvious, especially given the "want" declaration, so I just removed them.